policy/datasets.yaml

```diff
@@ -570,6 +570,16 @@ sourceTable_visit:
     python: lsst.pipe.tasks.parquetTable.ParquetTable
     tables: raw
     template: ''
+visitSummary:
+    description: >
+        Per-visit combination of visit metatadata, WCS, photoCalib, and
+        additional quantities.
+        Produced by ConsolidateVisitSummaryTask.
+    persistable: ExposureCatalog
+    storage: FitsCatalogStorage
+    python: lsst.afw.table.ExposureCatalog
+    tables: raw
+    template: ''
```

Comment:
I think this is something that could have a useful default template (yeah, it's gen2, but still). Maybe `summary/visitSummary-%(visit)07d.fits`? I don't understand why our parquet output doesn't have default templates either.

Reply:
Some of this is moot with the move to Gen3.  In the meantime, I'm going to follow the already established pattern.